README.md

```diff
@@ -1,64 +1,228 @@
-# GYFT - Get Your Freaking Timetable
+<div id="top"></div>
 
-Gets your timetable from ERP and adds it to your **Google Calendar** or gives you an **ICS file** which you can add in any common calendar application.
+<!-- PROJECT SHIELDS -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links-->
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+
+</div>
+
+<!-- PROJECT LOGO -->
+<br />
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/gyft">
+    <img width="140" alt="image" src="https://user-images.githubusercontent.com/86282911/206632284-cb260f57-c612-4ab5-b92b-2172c341ab23.png">
+  </a>
+
+  <h3 align="center">GYFT</h3>
+
+  <p align="center">
+  <!-- UPDATE -->
+    <i>Get Your Freaking Timetable</i>
+    <!-- <br /> -->
+    <!-- <a href="https://github.com/proffapt/PROJECT_NAME"><strong>Explore the docs »</strong></a> -->
+    <!-- <br /> -->
+    <br />
+    <a href="https://github.com/metakgp/gyft/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/metakgp/gyft/issues">Request Feature</a>
+  </p>
+</div>
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+<summary>Table of Contents</summary>
+
+- [About The Project](#about-the-project)
+  - [Supports](#supports)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Development Notes](#development-notes)
+- [Contributing](#contributing)
+- [Maintainer(s)](#maintainers)
+<!-- - [Contact](#contact) -->
+<!-- - [Acknowledgements](#acknowledgments) -->
+<!-- - [Additional documentation](#additional-documentation) -->
+
+</details>
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+<!-- UPDATE -->
+<!-- <div align="center">
+  <a href="https://github.com/proffapt/PROJECT_NAME">
+    <img width="80%" alt="image" src="https://user-images.githubusercontent.com/86282911/206632547-a3b34b47-e7ae-4186-a1e6-ecda7ddb38e6.png">
+  </a>
+</div> -->
+
+_Gets your timetable from ERP and adds it to your Google Calendar or gives you an ICS file which you can add in any common calendar application._
 
 > **Note** All updates to this repo should reflect, with appropriate refactorisation, in [gyft-serve](https://github.com/metakgp/gyft-serve/)
 
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-## Updates To Be Made Each Sem
+<div id="supports"></div>
 
-The following procedure is to be followed each new semester by the maintainer for updating GYFT to work properly.
+### Supports:
```

Comment:
This section can be either removed or mention the Python versions it supports and link to the webapp.

Reply:
Yes, removed all of it, and add the version of python in next section where pre-requisites are mentioned.